src/dispatch/conversation/flows.py

```diff
@@ -242,6 +244,62 @@ def set_conversation_topic(subject: Subject, db_session: SessionLocal):
         log.exception(e)
 
 
+def get_current_oncall_email(project: Project, service: Service, db_session: Session):
+    """Notifies oncall about completed form"""
+    oncall_plugin = plugin_service.get_active_instance(
+        db_session=db_session, project_id=project.id, plugin_type="oncall"
+    )
+    if not oncall_plugin:
+        log.debug("Unable to send email since oncall plugin is not active.")
+    else:
+        return oncall_plugin.instance.get(service.external_id)
+
+
+def get_description_text(subject: Subject, db_session: Session) -> str:
+    """Returns the descripion details based on subject"""
+    if isinstance(subject, Incident):
+        if subject.incident_type.channel_description is None:
+            return ""
+        if subject.incident_type.description_service is not None:
+            # add oncall email to description text, if applicable
+            oncall_email = get_current_oncall_email(
+                project=subject.project, service=subject.incident_type.description_service, db_session=db_session
+            )
+            if oncall_email:
+                return f"{subject.incident_type.channel_description.replace('{oncall_email}', oncall_email)}"
+        return f"{subject.incident_type.channel_description}"
+    return ""
```

Comment:
```suggestion
def get_description_text(subject: Subject, db_session: Session) -> str:
    """Returns the description details based on the subject"""
    if not isinstance(subject, Incident):
        return ""
    
    incident_type = subject.incident_type
    if not incident_type.channel_description:
        return ""
    
    description_service = incident_type.description_service
    if description_service:
        oncall_email = get_current_oncall_email(
            project=subject.project,
            service=description_service,
            db_session=db_session
        )
        if oncall_email:
            return incident_type.channel_description.replace("{oncall_email}", oncall_email)
    
    return incident_type.channel_description
```

Fail fast personal preference, I also think it may be better to either return `None` explicitly instead of an empty string or raise an `UnsupportedSubject` except and catch that.